base/changes.txt

```diff
@@ -1,4 +1,9 @@
-2020-04-21  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+2020-04-22 Ulrike Fischer<Ulrike.Fischer@latex-project.org>
```

Comment:
space before email address for consistency I guess

Reply:
The 2 spaces in front of my name or David's name are automatically placed there by the workflow we use in emacs (pressing C-c c and getting most of the line autoinserted in the right changes file), so removing it for consistency is probably counterproductive as 99% of the changes (at the moment are done by us 2 :-).  So I would rather add a space in front of Ulrike's name if we care about consistency

Reply:
Yep, that's kind of what I meant although I missed there were two spaces :)
Ulrike has now added a space before her address

Reply:
Well, she added one after her name but removed the second space(s) from the line with my name. Not that it matters to me, I was just explaining that whenever I generate such a line it will have those spaces in as they are auto-generated

Reply:
@FrankMittelbach  yes, sorry, some "autocorrecting space stuff" from my editor  kicked in (that was also what remove the one trailing space later one), but I didn't really realize that it would remove spaces when I touch the line. I readded both now.